test/nocks.ts

```diff
@@ -36,10 +37,9 @@ export function projectId(status: number|(() => string), reply?: () => string) {
     reply = status;
     status = 200;
   }
-  return nock('http://metadata.google.internal')
-      .get('/computeMetadata/v1/project/project-id')
-      .once()
-      .reply(status, reply, {'Metadata-Flavor': 'Google'});
+  return nock(HOST_ADDRESS).get(HOST_ADDRESS).once().reply(status, reply, {
```

Comment:
Are the two occurrences of `HOST_ADDRESS` intentional?

Reply:
Yikes. Thanks